docs/examples/testing.rst

```diff
@@ -425,8 +429,9 @@ connection from DTT on Line 4 and defines a Drawn Table with the initial state
 of the dimension in Line 5. By creating them in :meth:`.setUpClass()`, they are
 only initialized once and can be reused for each test. To ensure the tests do
 not affect each other, which would make the result depend on the execution
-order of the tests, the ``book`` table in the database is reset before each
-test by :meth:`.setUp()`. Then on Line 15 and Line 21, the tests are implemented
+order of the tests, the ``book`` table in the database is created and filled before each
+test by :meth:`.setUp()` and subsequently dropped after the test by :meth:`tearDown()`.
+Then on Line 15 and Line 21, the tests are implemented
```

Comment:
The line numbers should be updated to match the new location of the tests.

Reply:
Fixed